sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ScalaUDF.scala

```diff
@@ -1039,9 +1033,18 @@ case class ScalaUDF(
       (convert, argTerm)
     }.unzip
 
-    val callFunc = s"${ctx.boxedType(dataType)} $resultTerm = " +
-      s"(${ctx.boxedType(dataType)})${catalystConverterTerm}" +
-        s".apply($funcTerm.apply(${funcArguments.mkString(", ")}));"
+    val getFuncResult = s"$funcTerm.apply(${funcArguments.mkString(", ")})"
+    val rethrowException = "throw new org.apache.spark.SparkException" +
+      """("Exception happens when execute user code in Scala UDF.", e);"""
```

Comment:
Exception happens when `executing` user defined function (className:  input argument type => output argument type) .
Or 
`Failed to execute user defined function (className: input argument type => output argument type)